changelog/unreleased/pr-13018.toml

```diff
@@ -0,0 +1,92 @@
+values. Please see the sample [graylog.conf](https://github.com/Graylog2/graylog2-server/blob/master/misc/graylog.conf) file for more details and example values.
+
+- `elasticsearch_analyzer`
+- `elasticsearch_shards`
+- `elasticsearch_replicas`
+- `disable_index_optimization`
+- `index_optimization_max_num_segments`
+- `index_field_type_periodical_full_refresh_interval`
+- `rotation_strategy`
+- `elasticsearch_max_docs_per_index`
+- `elasticsearch_max_size_per_index`
+- `elasticsearch_max_time_per_index`
+- `retention_strategy`
+- `elasticsearch_max_number_of_indices`
+
+_Many of these `server.conf` configuration properties were un-deprecated as part of this feature, since Graylog intends
+to maintain them going forward._
+
+Once the first Graylog server instance is started to establish the cluster, the following system indexes will be created
+with the specified defaults.
+
+- Default index set
+- Graylog Events
+- Graylog System Events
+- Graylog Message Failures
+- Restored Archives
+
+## In-database Cluster Index Set Defaults
+
+The current in-database defaults for new Index Sets can now be edited from the new System > Configuration >
+Index Set defaults configuration area. The default values set here will be used for all new index sets created:
+
+- Those created from the System > Index Sets page.
+- New indexes created through the Graylog Illuminate system.
+
+Once Graylog 5.1 is installed, these in-database defaults will be established, and the server configuration option
+values described above will no longer be used.
+
+## New Index Default values
+Unless user-specified defaults are specified, the following new defaults will be effective for all new index sets created:
+
+- Shards: 1 (previously 4 in many cases)
+- Rotation Strategy: Index Size - 10GB (previously Index Time [1D] in many cases)
+"""
+
+details.ops = """
+The in-database Index Set default configuration can also be edited VIA the Graylog API:
+
+```
+curl 'http://graylog-server:8080/api/system/indices/index_set_defaults' \
+  -X 'PUT' \
+  -H 'Content-Type: application/json' \
+  -H 'X-Requested-By: my-api-request' \
+  --data-raw '
+  {
+     "index_analyzer":"standard",
+     "shards":5,
+     "replicas":5,
+     "index_optimization_max_num_segments":1,
+     "index_optimization_disabled":false,
+     "field_type_refresh_interval":28,
+     "field_type_refresh_interval_unit":"SECONDS",
+     "rotation_strategy_class":"org.graylog2.indexer.rotation.strategies.MessageCountRotationStrategy",
+     "rotation_strategy_config":{
+        "type":"org.graylog2.indexer.rotation.strategies.MessageCountRotationStrategyConfig",
+        "max_docs_per_index":10
+     },
+     "retention_strategy_class":"org.graylog2.indexer.retention.strategies.ClosingRetentionStrategy",
+     "retention_strategy_config":{
+        "max_number_of_indices":10,
+        "type":"org.graylog2.indexer.retention.strategies.ClosingRetentionStrategyConfig"
+     }
+  }'
+```
+"""
```

Comment:
Thanks for providing the API example.

I just discussed the example values with @bernd. The ones used here can be quite dangerous if people would apply this example 1:1. 

Could we just use the new defaults instead?

Reply:
Good catch, and good idea. I will fix that.